### PR TITLE
Nk/feature user data model

### PR DESCRIPTION
**Pull request by NatalyMac**
add fetching user data /user, /users, /users/id
{
  "data": {
    "first_name": "Paula",
    "last_name": "Schmidt",
    "email": "beverly.grimes@example.org",
    "city": "",
    "country": "",
    "birthday": "",
    "local_id": "",
    "avatar": {
      "urlAva": "",
      "thumbnailUrlAva": ""
    },
    "id": 331,
    "global_id": "567920fc215c82267c1a3cc3",
    "display_name": "jason63",
    "reputation": 212,
    "status_id": 19,
    "last_visit_at": "2015-10-01 07:33:29",
    "role_id": 19,
    "role": "User",
    "status": "online"
  },
  "_meta": []
}

change curl with get_file_content because of the trouble  with double  sending curl  - authentication fail from auth server

### CODE REVIEW

File: app/Providers/AppServiceProvider.php [hunk 1]
```diff
@@ -46,6 +46,8 @@ public function boot()
      */
     public function register()
     {
-        //
+        $this->app->bind(
+            'Repositories\Contracts\UserStoreInterface',
+            'Repositories\UserStore');
```
Comment (binaryvo) [inline app/Providers/AppServiceProvider.php]: Move closing parenthesis to next string please (psr-2)

**Comment by binaryvo** [comment]
Why have you changed permission modes for next files:
bootstrap/app.php
bootstrap/autoload.php
bootstrap/cache/.gitignore
?

**Comment by NatalyMac** [comment]
Why have you changed permission modes for next files
- sometimes I have Laravel fails with files permission - storage and bootstrap. And I've changed them. I will change them back

I've changed them to 644 and now I have 
warning: «bootstrap/.gitignore» недоступно: Отказано в доступе
bootstrap/app.php: Отказано в доступе
bootstrap/autoload.php: Отказано в доступе
bootstrap/cache/.gitignore: Отказано в доступе

when I try git add -A
what should I do?

File: app/Repositories/UserStore.php [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        $userProfile['id'] = $user->id;
+        $userProfile['global_id'] = $user->global_id;
+        $userProfile['display_name'] = $user->display_name;
+        $userProfile['reputation'] = $user->reputation;
+        $userProfile['status_id'] = $user->status_id;
+        $userProfile['last_visit_at'] = $user->last_visit_at;
+        $userProfile['role_id'] = $user->role_id;
+
+        $role = \DB::table('roles')->where('id', $user->role_id)->value('name');
+        $status = \DB::table('user_statuses')->where('id', $user->status_id)->value('name');
```
Comment (binaryvo) [inline app/Repositories/UserStore.php:55]: Please use models instead

File: app/Repositories/UserStore.php [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+                    $userInner = array_shift($userInner);
+
+                    $r['id'] = $userInner['id'];
+                    $r['display_name'] = $userInner['display_name'];
+                    $r['reputation'] = $userInner['reputation'];
+                    $r['status_id'] = $userInner['status_id'];
+                    $r['last_visit_at'] = $userInner['last_visit_at'];
+                    $r['role_id'] = $userInner['role_id'];
+
+                    $role = \DB::table('roles')->where('id', $r['role_id'])->value('name');
```
Comment (binaryvo) [inline app/Repositories/UserStore.php:97]: Please use models instead

File: app/Repositories/UserStore.php [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+                    $r['first_name'] = $userFunc->name;
+                    $r['last_name'] = $userFunc->surname;
+                    $r['email'] = $userFunc->email;
+                    $r['city'] = $userFunc->city;
+                    $r['country'] = $userFunc->country;
+                    $r['birthday'] = $userFunc->birthday;
+                    $r['local_id'] = $userFunc->id;
+                    $r['avatar'] = $userFunc->avatar;
+                    return $r;
+                },
```
Comment (binaryvo) [inline app/Repositories/UserStore.php:28]: I guess it could be possible to move mapper to separate function to prevent repetitive code

**Comment by binaryvo** [comment]
Please fix all remarks and make PR again

**Pull request closed without merging**